submission/submission-webapp/src/itest/resources/application.yml

```diff
@@ -43,9 +43,11 @@ app:
   security:
     tokenHash: biostd-test
     environment: TEST
-    filesDirPath:
-    magicDirPath:
     requireActivation: false
+    filesProperties:
+      filesDirPath:
```

Comment:
I think we could use this opportunity to add documentation for the properties

Reply:
they have, but not in the itest file but in application yml files of reulgar application